src/react-components/src/components/02-molecules/Cards/EditionCard.stories.tsx

```diff
@@ -62,16 +64,26 @@ export const editionCardWithEmptyEditionInfo = () => <EditionCard
   coverUrl="https://placeimg.com/300/400/arch"
   editionHeadingElement={<a href="edition-link" className={bem("link", [], "heading")} >2004 Edition</a>}
   editionInfo={[]}
-  readOnlineLink="#readOnlineUrl"
-  downloadLink="#downloadUrl">
+  readOnlineLink={<BasicLink className={bem("card-button-link", [], "edition-card")} url="blah">Read Online</BasicLink>}
+  downloadLink={<IconLink iconName="download" iconPosition="left" url="blah">Download</IconLink>}>
 </EditionCard>;
 
 export const editionCardWithAtypicalImageSize = () => <EditionCard
```

Comment:
I like this story. It makes me think that there should be a background on these images, such as [catalog item](http://themetronome.co/components/mmb-4-indi-catalog-item/?fresh=true). (Catalog item is using `$gray-warm-light` for its background, I believe.)

Reply:
I'd wait till after the card consolidation work to apply this - for now, I think the metronome doesn't reflect this and it might confuse QA on our end.